apex/contrib/csrc/groupbn/cuda_utils.h

```diff
@@ -1,4 +1,8 @@
+#ifdef __HIP_PLATFORM_HCC__
+#include <ATen/hip/HIPContext.h>
+#else
 #include <ATen/cuda/CUDAContext.h>
+#endif
 #ifndef CUDA_UTILS_H
```

Comment:
This ifdef change is not needed since hipify will do this replacement in the generated hip_utils.h file